AndroidIDPTemplate/app/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 dependencies {
-  implementation project(':libs:MobileSync')
+    implementation 'com.salesforce.mobilesdk:MobileSync:11.0.0'
```

Comment:
If we do it this way it won't dev version of the template won't use the dev version of the Mobile SDK and we won't be able to detect regressions until we tag the release.

Reply:
Is the idea to do:
- get lib from maven central in production 
- get lib through a composite build in dev
In that case we should tweak our setversion.sh / install.js / settings.gradle files between prod and dev.
In prod, we should omit the `includeBuild(new File(settingsDir, 'mobile_sdk/SalesforceMobileSDK-Android' from settings.gradle and not clone the SalesforceMobileSDK-Android repo in install.js.